source/wp-content/themes/wporg-developer-2023/src/style/style.scss

```diff
@@ -14,3 +14,68 @@
 .wporg-filtered-search-form {
 	display: block;
 }
+
+.wp-block-search.wp-block-search__button-inside .wp-block-search__inside-wrapper {
```

Comment:
This was brought over from documentation rebuild to stack the columns on mobile.